pbuilder-hookdir/B90lintian

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+#
+# Hook to run lintian inside the cowbuilder environment.
+#
+
+BUILDDIR="${BUILDDIR:-/tmp/buildd}"
+
+if [ "${LINTIAN:-}" != "true" ] ; then
+  echo "Skipping lintian (LINTIAN is not 'true')"
+  exit 0
+fi
+
+if [ -n "${LINTIAN_OPTIONS:-}" ] ; then
+  echo "*** Using provided LINTIAN_OPTIONS $LINTIAN_OPTIONS ***"
+fi
+
+set -ex -o pipefail
+apt-get -y "${APTGETOPT[@]}" install lintian
+lintian --version
+
+package_dir=$(cd "$BUILDDIR"/*/debian/.. 2>/dev/null && pwd -P)
+PACKAGE_NAME=$(basename "$package_dir")
```

Comment:
Updated to no more rely on realpath. Deployed on Wikimedia infrastructure and it works! https://phabricator.wikimedia.org/T193906